src/draw/DefaultDrawer.js

```diff
@@ -21,13 +22,20 @@ class DefaultDrawer {
    * @param {number[]} [options.lineLengthPerDepth] - Number of components
    * per line at a given depth. Valid values: 1 - Infinity.
    * @param {number} [options.actionMenuButtonSize] - The size of each action menu button.
+   * @param {DefaultLayout} [layout] - The manager for an automatic diagram layout
    */
-  constructor(pluginData, resources = null, rootId = 'root', options = {}) {
+  constructor(pluginData, resources = null, rootId = 'root', options = {}, layout = undefined) {
```

Comment:
`undefined` is already the default value, so no need to specify it.

Reply:
done

Reply:
We cannot have a non-optional argument (ie. without an explicit default value) after an optional argument.
(`ESLint: Default parameters should be last.(default-param-last)`)
Moving `layout` as the second argument would be a breaking change.

I suggest to leave it as it is.
Do you agree ?

Reply:
Set `null` instead as`resources`

Reply:
done